common/FileUtil.cpp

```diff
@@ -234,11 +234,24 @@ namespace FileUtil
     std::string getTempFileCopyPath(const std::string& srcDir, const std::string& srcFilename, const std::string& dstFilenamePrefix)
     {
         const std::string srcPath = srcDir + '/' + srcFilename;
-        const std::string dstFilename = dstFilenamePrefix + Util::encodeId(Util::rng::getNext()) + '_' + srcFilename;
```

Comment:
Breaks the !HAVE_STD_FILESYSTEM case which seems to be that of our current android/iOS ci build. I'll update this pr to fix that.